src/test/java/bio/terra/service/snapshot/SnapshotConnectedTest.java

```diff
@@ -561,4 +759,30 @@ private long queryForCount(
         return countValue.getLongValue();
     }
 
+    private String getFileRefIdFromSnapshot(SnapshotSummaryModel snapshotSummary) throws InterruptedException {
```

Comment:
I like pulling some of this BigQuery stuff out into a helper method, but I think it's very specific to your particular test. I wonder if you could factor out the snapshot, table and column names to make it more general? And maybe also the where/limit clause? If that's too much work, then I think you could also just put this inline into the test.